TheJumpCloud/support
# rename local user automation

## Issues
* [SA-2871](https://jumpcloud.atlassian.net/browse/SA-2871) - Rename Local Users from CSV

## What does this solve?

Adds a customer requested automation to our support pages repository.

## Is there anything particularly tricky?

## How should this be tested?

Prepare two VMs, windows & mac, create a new local user in each system.

Run the powershell scripts, a CSV should be created / uploaded to a public location (private gist location is fine). Identify the ID of the two VM systems and manually (or if already pulled in via the powershell automations) add the userToMigrate/ jumpcloud usernames to the CSV. 

Run the corresponding scripts on the Windows & Mac hosts. 

In windows, the user should be "renamed" to the jumpcloud user and taken over. log out/ login and the user's Full Name should be updated.

In Mac, any user logged into the system will be logged out. The login window should "Refresh" (black screen for a moment) and the local user should be taken over — their username should be updated and when logging in, the old password/ new password prompt should display. Login, the dock should be reset the the default settings. Upon a subsequent login/ restart the desktop should load normally.


## Screenshots
